ontology/yaml/resources/LIGHTING/entity_types/LCM.yaml

```diff
@@ -21,3 +21,4 @@ LCM_SS:
   implements:
   - LCM
   - SS
+
```

Comment:
revert if just whitespace.

Reply:
Yes, It is a white space and has been removed...